crates/rolldown_common/src/module/normal_module.rs

```diff
@@ -85,6 +89,28 @@ impl NormalModule {
       },
       imported_ids: self.ecma_view.imported_ids.clone(),
       dynamically_imported_ids: self.ecma_view.dynamically_imported_ids.clone(),
+      // https://github.com/rollup/rollup/blob/master/src/Module.ts#L331
```

Comment:
use permalink in case rollup code changes?
```suggestion
      // https://github.com/rollup/rollup/blob/7a8ac460c62b0406a749e367dbd0b74973282449/src/Module.ts#L331
```